app/submitter/converter.py

```diff
@@ -186,6 +193,16 @@ def convert_answers_to_census_data(answer_store, routing_path):
     return data
 
 
+def format_dates(answer_type, value):
+    if answer_type == 'Date':
+        return datetime.strptime(value, '%Y-%m-%d').strftime('%d/%m/%Y')
+
+    if answer_type == 'MonthYearDate':
+        return datetime.strptime(value, '%Y-%m').strftime('%m/%Y')
```

Comment:
Is there value in having this method? We could just do this formatting in `_format_downstream_answer`